added experimental release note for view transition api (#39588)
mdn/content: files/en-us/mozilla/firefox/experimental_features/index.md

  </tbody>
</table>

### View Transition API

> The [View Transition API](/en-US/docs/Web/API/View_Transition_API) has been enabled for [SPAs (Single-page Applications)](/en-US/docs/Glossary/SPA) in the Nightly release. This provides a mechanism for easily creating animated transitions between different website views. ([Firefox bug 1950759](https://bugzil.la/1950759)).

Reviewer: Maybe we can drop "has been enabled in the Nightly release" because we're conveying that through the table?
And so we could say:
```suggestion
The [View Transition API](/en-US/docs/Web/API/View_Transition_API) provides a mechanism for easily creating animated transitions between different website views. This is especially useful for [SPAs (single-page applications)](/en-US/docs/Glossary/SPA). ([Firefox bug 1950759](https://bugzil.la/1950759)).
```
